src/components/procedure/_pr-comments-in-secrets.mdx

```diff
@@ -0,0 +1,46 @@
+<span>
+  In addition to setting up the connection between Semgrep and {props.name}, you
+  must assign rules to Comment or Block mode. This customization enables you to:
+</span>
+
+- Manage the <strong>amount of {props.comment_type} comments</strong> your
+    developers receive.
+- Ensure that only rules that meet your criteria, such as high severity or high confidence rules, produce comments visible to developers, reducing noise.
+
+#### Set rules to Comment or Block mode
+
+The following instructions let you customize what findings or security issues your developers see as comments in their PRs:
+
+1. In Semgrep AppSec Platform, go to **Rules > Policies > Secrets**.
+1. Under **Modes <i class="fa-solid fa-gear"></i>**, you can see if you have existing rules in either Comment or Block mode. You can also use the filters to find rules you want to set to Comment or Block.
+1. Click the **<i class="fa-solid fa-square-check"></i> checkbox** of the rules you want to set. You can use <kbd>Ctrl + Click</kbd> to select rules in bulk.
+1. Click **Change modes**.
+1. Click either **Block** or **Comment**.
+
+<span>
+  You have successfully configured {props.comment_type} comments for Semgrep
+  Secrets.
+</span>
+
+#### Validation state policies
+
+Validation state policies allow you to define how Semgrep handles the following issues:
+
+- **Invalid findings**: the secret has been revoked or was never functional
```

Comment:
was never functional, or is for a custom/private endpoint that Semgrep does not communicate too.